### DISCUSSION TITLE
Formatting broken displaying email issue comments on Mobile

### DISCUSSION THREAD

**Discussion by citizenmatt** [community/community#43049] [Mobile] [answered]
Hi folks. The formatting for comments added by email is broken on GitHub mobile. Instead of showing quoted sections, everything is shown as a single block of text, which makes the comments hard to read. They are shown correctly on github.com.

An example can be seen in this PR: https://github.com/vim/vim/pull/11764

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone15,2 (iOS 16.2)
</details>

**Comment by HenSquared** [comment, marked as answer]
Great find. 

I was able to reproduce this on my iOS device and it is definitely hard to read without correct formatting. I'm going to open an issue to get this feedback addressed.

Thanks for the feedback!

**Reply by citizenmatt** [reply]
Great, thanks!

**Reply by citizenmatt** [reply]
Just spotted that this was fixed a little while ago. Thanks! 👍
